content/guides/monitoring.md

```diff
@@ -347,11 +390,13 @@ right people get notified so the problem can be resolved as soon as possible.
 
 - *Can you alert on a function?*
 
-  Not currently but this feature is in the works.
+  Yes, with a feature called 'query alerts'. Selecting the 'Source' tab of a
```

Comment:
I'd be in favor of not mentioning 'query alerts' at all as the user doesn't really need to care about that name. So just say "Yes, selecting the 'Source' tab..."